cmd/prepare-storage-msg.go

```diff
@@ -51,12 +51,6 @@ func printOnceFn() printOnceFunc {
 	}
 }
 
-// Prints custom message when healing is required for XL and Distributed XL backend.
-func printHealMsg(endpoints []*url.URL, storageDisks []StorageAPI, fn printOnceFunc) {
```

Comment:
Actually we should print the heal message with the `mc` command what needs to be done.